### ISSUE TITLE
Dead corpses more AoS-like

### ISSUE THREAD

**Issue by Clogal** [yvt/openspades#645]
Since Ragdolls looks really awesome and all, this probably would get ignored, but i prefer the old aos dead corpses, actually there exist but needs some improvements like that bouncing effect, it's not a big thing, but would be really nice having them on OpenSpades.

**Comment by NotAFile** [comment]
Hi, thanks for reporting this!

You have been able to turn ragdolls off in the settings for a while now. So what you are asking is if we could add the bouncing effect?

I guess there is nothing wrong with adding that, but I don't think this will be a priority so you might have to wait for a long time.

**Comment by Clogal** [comment]
Yes, it's not a priority sorry, it just need some improvements like i said, in my opinion that would be adding bouncing effect, now all death corpses(not-ragdolls) faces north wherever you shot at, anyways im going to wait for a long time, hope this get implemented, thanks!

**Comment by yvt** [comment]
Could you provide a link to a footage showing how the old AoS corpses look like? I haven't played the original version for some long time.

**Comment by Clogal** [comment]
Yes, video : [https://www.youtube.com/watch?v=eqaPMV5YJqw](url)

**Comment by yvt** [comment]
@Clogal Thanks! I'll get on it.
